traffic_monitor/towrap/towrap.go

```diff
@@ -277,8 +312,30 @@ func (s TrafficOpsSessionThreadsafe) trafficMonitorConfigMapRaw(cdn string) (*tc
 	if ss == nil {
 		return nil, ErrNilSession
 	}
-	configMap, _, error := ss.GetTrafficMonitorConfigMap(cdn)
-	return configMap, error
+	var configMap *tc.TrafficMonitorConfigMap
+	var err error
+
+	configMap, _, err = ss.GetTrafficMonitorConfigMap(cdn)
```

Comment:
Nitpick: don't need to pre-declare, can remove the `var`s and `configMap, _, err := ss.GetTrafficMonitorConfigMap(cdn)`